<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GiveMoneyDialog</class>
 <widget class="QDialog" name="GiveMoneyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>充值</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <!-- 充值界面 -->
     <widget class="QWidget" name="chargePage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="topBar" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>充值中心</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="contentWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QLabel" name="balanceLabel">
            <property name="text">
             <string>当前余额：¥ 0.00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="amountWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>选择充值金额：</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <property name="spacing">
                <number>15</number>
               </property>
               <item row="0" column="0">
                <widget class="QPushButton" name="amount_10">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>¥10</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="amount_20">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>¥20</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="amount_50">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>¥50</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="amount_100">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>¥100</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="amount_200">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>¥200</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="amount_500">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>¥500</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="customAmountEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>45</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>输入自定义金额</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="paymentWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>选择支付方式：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="wechatPayBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>微信支付</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="alipayBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>支付宝</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <!-- 扫码界面 -->
     <widget class="QWidget" name="scanPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="scanTopBar" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scanTitleLabel">
            <property name="text">
             <string>扫码支付</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>35</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="preview_label">
         <property name="minimumSize">
          <size>
           <width>497</width>
           <height>693</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scan_line">
         <property name="minimumSize">
          <size>
           <width>670</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>670</width>
           <height>2</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
